docs/rules/forbid-component-props.md

```diff
@@ -35,10 +37,94 @@ The following patterns are not considered warnings:
 
 ```js
 ...
-"forbid-component-props": [<enabled>, { "forbid": [<string>] }]
+"forbid-component-props": [<enabled>, { "forbid": [<string>|<object>], "defaultAllowOnComponents": <bool>, "defaultAllowOnDOM": <bool>}]
```

Comment:
the two "default" booleans need to be on the object, not on the outer config object. not sure how to denote this tho.